.github/workflows/publish-kentik.yml

```diff
@@ -0,0 +1,74 @@
+name: Publish Kentik
```

Comment:
This file is almost an exact duplicate of `publish-staging.yml` (and quite similar to other `publish-*.yml` files). At some point it would be good to refactor and reduce duplication.

Reply:
Hah! This is a very valid point. Probably can combine into some kinda action but man do I hate writing this stuff.